docs/os/tutorials/blinky_console.md

```diff
@@ -180,26 +176,24 @@ Once you have a connection set up, you can connect to your device as follows:
 To test and make sure that the Shell is running, first just hit <return>:
     
 ```no-highlight
-3534: >
+004543 shell>
 ```
 
 You can try some commands:
 
 ```no-highlight
-3609: > ?
-Commands:
-8841:     echo         ?    prompt     ticks     tasks  mempools
-8843:     date         b
-8844: > ticks off
- Console Ticks off
- > prompt off
- Prompt now off.
-ticks on
-33383: Console Ticks on
-
-33568:
-prompt on
-39108: Prompt now on.
-39108: >
+014930 shell> help
+015242 Available modules:
+015242 os
+015243 To select a module, enter 'select <module name>'.
+015244 shell> os <TAB>
+015525 help
+015525 tasks                         show os tasks
+015526 mpool                         show system mpool
+015526 date                          show system date
+015527 shell> os date
+016923 module: 0, command: date
+016924 1970-01-01T00:02:12.218736+00:00
+016925 shell>
 ```
```

Comment:
I think the confusing thing for me is that:  In the old shell/console implementation, the old shell tick command controlled the tick that was printed out by console. So if you had ticks off then you wouldn't see any ticks in the prompt at all. Now the console tick is controlled by syscfg only. And the ticks command in the new shell actually adds another tick prompt to it. So the output looks like this now: 
```
007953 prompt> ticks on                                                         
008517  Console Ticks on                                                        
008517 8517> > ticks off                                                        
042377  Console Ticks off                                                       
042377 prompt> ticks on                                                         
043620  Console Ticks on                                                        
043620 43620> > 
```
So I thought it would be good to explain that in the tutorial.